### ISSUE TITLE
Adapting eqheight

### ISSUE THREAD

**Issue by glupto** [jsliang/eqHeight.coffee#10]
Hi, there. I try to relaunch my website that loads a lot of feeds. Therefore I've used (very thankfully) the eqheight-script. My website is loaded after every feed has been fetched by php. Now I try to adapt eqheight for a js-solution, that loads the first feeds by a request, than shows the website and loads all the other feeds in background. But now I don't know where to integrate the eqheight script. The way I tried it (without knowing a lot about js) does not work. If I load the functions at the end of my html, it does not work, because the feeds aren't still there. It only works when I resize.

Here' my code:

[CODE]  <div class="main"  style="margin: 60px 0px 10px 10px;">
          

  <script type="text/javascript">

var feedObj = null;
    feedObj = new XMLHttpRequest();
var i = 1;

window.onload = function() {

```
function beginneLaden() {

    feedObj.open("GET", "feeds1.php?feedid="+i);
    feedObj.onreadystatechange = ladeFeeds;
    feedObj.send(null);
    $(".main").eqHeight("article", {equalize_interval: 500});

    function ladeFeeds() {

        var fa = false;
        var anzahl = 18;      // Anzahl der Feeds angeben

        if (feedObj.readyState == 4) {
            document.getElementById("feed"+i).innerHTML += feedObj.responseText;
            i++;
            fa = true;
            if (i > anzahl) {fa = false;
            }
        }
        if (fa != false) {

            beginneLaden();

        }
    }
}
beginneLaden();
```

$(".main").eqHeight("article", {equalize_interval: 500});

$(window).resize(function(){
  equalheight('.main article');
});

}
</script>

   <article id="feed1"> 
               <a href="http://www.tagesschau.de" target='_blank'> 
            <img src="../logos/tagesschau.jpg" alt="Tagesschau" class="img-rounded"/></a>          

</article>

  <article id="feed2"> 
  <a href="http://www.deutschlandfunk.de/nachrichten.353.de.html" target='_blank'> 

```
                   <img src="logos/dlf.jpg" alt="Deutschlandfunk" class="img-rounded"/></a>
```

 </article>

  <article id="feed3"> 
  <a href="http://www.spiegel.de/" target='_blank'> 
  <img src="logos/spiegel.png" alt="Der Spiegel" class="img-rounded"/></a>

 </article>
[/CODE]
 .....and so on....

COuld you help be adapting eqheight in the right way. The equalizing function should start after the first feed(s) is/are loaded. Here's the well working URL for the php-based site (before relaunch)  [URL]http://nachrichtentisch.de/test.php[/URL]

And here is the only-by-resizing-working solution for the js-based relaunch:
[URL]http://nachrichtentisch.de/test1.php[/URL]

Thansk for helping!!!
Kind regards 
Peter

**Comment by jsliang** [comment]
Hi Peter,

I inspected the source of test1.php and found that the function `eqHeight` is not defined in your page. (I noticed that there is a similar script `scripts2.js`, but the function names defined in this script are different from those in eqHeight.) It looks to me that you need to import the `jquery.eqheight.js` script for this purpose:
`<script src="jquery.eqheight.js"></script>` (be sure to put this line after importing jQuery)

Kindly let me know if this still doesn't work. :)

jsliang

**Comment by glupto** [comment]
Hi, Jenny, 

thank you for answering so immediately.

I imported the jquery.eqheight.js  into my 
"nachrichtentisch.de/test1.php"-site and it works! I'm astonished, that my 
old script "nachrichtentisch.de/test.php" worked without that and only 
with my "scripts2.js". To be honest, I already try to relaunch my site for 
months now and don't know, when I put the equalheight function together 
with two other js-functions into this "scripts2.js". And I don't know 
what's the difference between the equalheights- and the 
eqheights-function.

Now anyway - it works, as I said, but only if I use it together with the 
option {{equalize_interval: 500}  ... or 1000, because the feeds are 
loaded one after another and the first one doesn't know how long the next 
one will be, I guess. 

And: If I try to look at the last feed (a stray feed in a row), the site 
jumps up some centimeters. But maybe this is a css-Problem, don't know.

If you have got an idea about that, I would be thankful, if you write 
again.

Now I try to bring my tooltipster-plugin to work.

Kind regards
Peter
(linving in Berlin, Germany)

Von:    "Jui-Shan Liang (Jenny)" notifications@github.com
An:     "jsliang/eqHeight.coffee" eqHeight.coffee@noreply.github.com
Kopie:  glupto peter.tomuscheit@rbb-online.de
Datum:  24.05.2014 10:46
Betreff:        Re: [eqHeight.coffee] Adapting eqheight (#10)

Hi Peter,
I inspected the source and test1.php and found that the function eqHeight 
is not defined in your page. (I noticed that there is a similar script 
scripts2.js, but the variable names are different from eqHeight.) It looks 
to me that you need to import the jquery.eqheight.js script for this 
purpose:
<script src="jquery.eqheight.js"></script> (be sure to put this line after 
importing jQuery)
Kindly let me know if this still doesn't work. :)
jsliang
?
Reply to this email directly or view it on GitHub.

Peter Tomuscheit
Redakteur

Rundfunk Berlin-Brandenburg (rbb)
Inforadio
Masurenallee 8-14
14057  Berlin
Telefon: +49 30 97993 37110
Telefax: +49 30 97993 37139
peter.tomuscheit@rbb-online.de      

www.inforadio.de | www.rbb-online.de

Ihr Rundfunkbeitrag für gutes Programm.

**Comment by glupto** [comment]
Dear Jenny, sorry for bothering you again with a question.

I have to quit my plans with a slidebar-Menu, because the programmer quit 
his project, because he finds there are too many errors in it. (though I 
couldn't find them). Now I try to find something new and try to use this 
http://tympanus.net/Development/MultiLevelPushMenu/.

Could you please look again at my scripts. The old one works fine with 
eqheight, but the new one mit the multilevel thing does not, though I 
compared both codes (And I'm certain this time, that I built in 
jquery.eqheight.js" at the right place.

Works: http://nachrichtentisch.de/test2.php

Does not work: http://nachrichtentisch.de/collapsed.php

Thank you a lot!

Kind regards from Berlin
Peter

Von:    "Jui-Shan Liang (Jenny)" notifications@github.com
An:     "jsliang/eqHeight.coffee" eqHeight.coffee@noreply.github.com
Kopie:  glupto peter.tomuscheit@rbb-online.de
Datum:  24.05.2014 10:46
Betreff:        Re: [eqHeight.coffee] Adapting eqheight (#10)

Hi Peter,
I inspected the source and test1.php and found that the function eqHeight 
is not defined in your page. (I noticed that there is a similar script 
scripts2.js, but the variable names are different from eqHeight.) It looks 
to me that you need to import the jquery.eqheight.js script for this 
purpose:
<script src="jquery.eqheight.js"></script> (be sure to put this line after 
importing jQuery)
Kindly let me know if this still doesn't work. :)
jsliang
?
Reply to this email directly or view it on GitHub.

Peter Tomuscheit
Redakteur

Rundfunk Berlin-Brandenburg (rbb)
Inforadio
Masurenallee 8-14
14057  Berlin
Telefon: +49 30 97993 37110
Telefax: +49 30 97993 37139
peter.tomuscheit@rbb-online.de      

www.inforadio.de | www.rbb-online.de

Ihr Rundfunkbeitrag für gutes Programm.

**Comment by jsliang** [comment]
Hi Peter,

I look at http://nachrichtentisch.de/collapsed.php , and this page works for me. The cells are stretched to equal heights as expected. Did you fix that earlier? :D

**Comment by glupto** [comment]
Dear Jenny, thanks for your answer. Yes I fixed it by changing the id=main 
to a class=main. I fixed just after I wrote the mail to you and I called 
the mail back (that works sometimes with lotus, but it seems it did not 
work this time). Now most of the things work except my footer, but I will 
solve this. Sorry for bothering you.

Kind regards, Peter

Von:    "Jui-Shan Liang (Jenny)" notifications@github.com
An:     "jsliang/eqHeight.coffee" eqHeight.coffee@noreply.github.com
Kopie:  glupto peter.tomuscheit@rbb-online.de
Datum:  12.06.2014 16:01
Betreff:        Re: [eqHeight.coffee] Adapting eqheight (#10)

Hi Peter,
I look at http://nachrichtentisch.de/collapsed.php , and this page works 
for me. The cells are stretched to equal heights as expected. Did you fix 
that earlier? :D
?
Reply to this email directly or view it on GitHub.

Peter Tomuscheit
Redakteur

Rundfunk Berlin-Brandenburg (rbb)
Inforadio
Masurenallee 8-14
14057  Berlin
Telefon: +49 30 97993 37110
Telefax: +49 30 97993 37139
peter.tomuscheit@rbb-online.de      

www.inforadio.de | www.rbb-online.de

Ihr Rundfunkbeitrag für gutes Programm.
